includes/functions.php

```diff
@@ -2080,86 +2074,88 @@ function tsml_import_has_changes($data_source_url, $meetings, $data_source_count
 		}
 	}
 
-	// create array of db slugs for matching
+	// create array of database slugs for matching
 	$db_slugs = array_column($db_meetings, 'slug', 'id');
 	sort($db_slugs);
-	$db_slugs_cnt = count($db_slugs);
 
-	// list changed and new meetings found in the ds feed
-	foreach ($meetings as $meeting) {
+	// list changed and new meetings found in the data source feed
+	foreach ($feed_meetings as $meeting) {
+	
+		$dayofweek = $dow_number = '';
+		tsml_get_day_of_week_info( $meeting['day'], $dayofweek, $dow_number);
+	    $meeting_slug =  $meeting['slug'];
 
-	    $mtg_slug =  $meeting['slug'];
-		if (is_array($meeting['day'])) {
-			$dow = implode("",  $meeting['day']);
-		} else {
-			$dow = $meeting['day'];
-		}
-		
-		if ( is_numeric( $mtg_slug )) {
-			$mtg_slug .= '-' . $dow; 
+		// numeric slugs may need some reformatting
+		if ( is_numeric( $meeting_slug )) {
+			$meeting_slug .= '-' . $dow_number; 
 		}
 
 		// match feed/database on unique slug
-	    $is_matched = in_array( $mtg_slug, $db_slugs );
+	    $is_matched = in_array( $meeting_slug, $db_slugs );
 
 		// add slug to feed array to help determine current db removals later on...
-		$feed_slugs[$meeting['slug']] = $mtg_slug;
-
-		// has meeting been updated?
-		$updated = $meeting['updated'];
-		$cur_meeting_lastupdate = strtotime($updated);
-		if ($cur_meeting_lastupdate > $data_source_last_refresh) {
-			$is_there_change = true;
-			$mtg_name = $meeting['name'];
-			$mtg_updte = date("M j, Y  g:i a", $cur_meeting_lastupdate);
-			if ( is_numeric( $dow )) {
-				$dayofweek = $days[date($dow)]; 
-			} else {
-				$dayofweek = $dow; 
-			}
+		$feed_slugs[$meeting['slug']] = $meeting_slug;
```

Comment:
this doesn't need to be associative, so i think you just mean
```
$feed_slugs[] = $meeting_slug;
```

Reply:
I think your right...